css/style.css

```diff
@@ -43,6 +42,121 @@ body {
   margin: auto;
   height: 500px;
 }
+.logoOne {
+  margin-left: 100px;
+  font-size: 24px;
+  font-weight: 900;
+  color: white;
+}
+.message {
+  text-align: center;
+  font-weight: bold;
+  font-size: 24px;
+  color: white;
+  margin-top: 150px;
+}
+.centerButton { 
+  text-align: center;
+  margin-top: 50px;
+}
+.getIt {
+  background-color: #f9e42e;
+  font-size: 14px;
+  font-weight: 900;
+  color: #121212;
+  border-radius: 5px;
+  padding: 10px 20px;
+}
+.greyBackground {
+  width: 100%;
+  height: 280px;
+  background-color: #f0efef;
+}
+.benefits {
+  margin-left: 100px;
+  font-weight: bold;
+  font-size: 20px;
+  color: #033048;
+  padding-top: 30px;
+}
+.aboutRelaxr{
+  margin-left: 100px;
+  font-weight: lighter;
+  font-size: 13px;
+  color: #606161;
+}
+ul { /*how do i chance bullet color and size */
```

Comment:
Using normal bullets (as you're doing here), the bullet color controlled the same way text is - by setting `color`. If you want them to be styled different, there's some other techniques you can use that are a bit more advanced. if you'd like to discuss, stop by office hours and I can show you a technique.